### PR TITLE
Review of plugin-api-migration.md:

### PR DESCRIPTION
**Pull request by gaaclarke**
- annotated code samples
- minor formatting error

relevant ticket: https://github.com/flutter/flutter/issues/44224

### CODE REVIEW

File: src/docs/development/packages-and-plugins/plugin-api-migration.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ and redirect to this file.
```
Comment (xster) [inline src/docs/development/packages-and-plugins/plugin-api-migration.md]: we should qualify that this is for Android in the permalink, title etc.

File: src/docs/development/packages-and-plugins/plugin-api-migration.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ and redirect to this file.
 
 {{site.alert.note}}
   You might be directed to this page if the framework detects that
-  your app uses a plugin based on the old APIs. 
+  your app uses a plugin based on the old APIs.
 {{site.alert.end}}
 
 If you don't write or maintain a Flutter plugin,
```
Comment (xster) [inline src/docs/development/packages-and-plugins/plugin-api-migration.md]: Android here too

File: src/docs/development/packages-and-plugins/plugin-api-migration.md [hunk 3]
```diff
@@ -134,7 +130,7 @@ public class EmbeddingV1Activity extends FlutterActivity {
 8. To have the plugin support Flutter on branches master and stable,
```
Comment (xster) [inline src/docs/development/packages-and-plugins/plugin-api-migration.md]: we can remove this whole step by the time stable is out.

**Comment by xster** [comment]
LGTM

File: src/docs/development/packages-and-plugins/plugin-api-migration.md (hunk 1 above)
Comment (gaaclarke) [reply src/docs/development/packages-and-plugins/plugin-api-migration.md]: done

File: src/docs/development/packages-and-plugins/plugin-api-migration.md (hunk 2 above)
Comment (gaaclarke) [reply src/docs/development/packages-and-plugins/plugin-api-migration.md]: done

File: src/docs/development/packages-and-plugins/plugin-api-migration.md (hunk 3 above)
Comment (gaaclarke) [reply src/docs/development/packages-and-plugins/plugin-api-migration.md]: lets revisit once the dust has settled

**Pull request merged**